src/components/home/Mission.tsx

```diff
@@ -6,15 +6,15 @@ const Mission = () => {
     <>
       <div className="flex w-full justify-between">
         <div className="my-5 w-3/4 flex-col p-12">
-          <div className="text-left font-songMyung text-5xl text-hsa-blue-100">
+          <div className="text-left font-songMyung text-5xl text-hsa-blue-100 md:text-center">
             Our Mission
           </div>
-          <div className="-ml-4 mr-5 p-5 text-left font-openSans text-xl font-semibold text-hsa-gray-300">
+          <div className="-ml-4 mr-5 p-5 text-left font-openSans text-xl font-semibold text-hsa-gray-300 md:text-center">
             Here at HSA @UCR, our mission is to spread knowledge of our history
             & culture, and to build a safe & uplifting Hmong community
           </div>
         </div>
-        <div className="relative flex">
+        <div className="relative flex md:flex">
```

Comment:
the md:flex doesnt do anything sense flex is already there
```suggestion
        <div className="relative flex">
```